pkg/controller/util.go

```diff
@@ -479,7 +479,7 @@ func MakeCloneTargetPodSpec(image, verbose, pullPolicy string, pvc *v1.Persisten
 									},
 								},
 							},
-							Namespaces: []string{podAffinityNamespace},
+							Namespaces:  []string{podAffinityNamespace},
```

Comment:
super nit: did `gofmt` add the trailing space?

Reply:
yes